lib/schema.js

```diff
@@ -183,7 +184,14 @@ function generateColumn(definition) {
   if (['integer', 'string'].indexOf(column.type) > -1 && size !== null) {
     column.size = parseInt(size[1]);
   }
-
+  if (definition.column_default === 'CURRENT_TIMESTAMP') {
+    column.defaultsTo = 'function() {return new Date();}';
+    column.required = false;
```

Comment:
Something can default to something, and still be required. We should use the "IS_NULLABLE" for that (which is already happening up ^). I'd refactor this set to:

```js
if (definition.column_default !== null) {
  let columnDefault = definition.column_default;
  column.defaultsTo = columnDefault === 'CURRENT_TIMESTAMP' ? 'function() {return new Date();}' : columnDefault;
}
```